tests/read_only_trx_test.py

```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env python3
```

Comment:
This test does not produce hardly any load on nodeos. It would be good to create a test that uses the `trx_generator`  to send regular and read-only trxs to nodeos at high load for stability testing. Maybe a useful task to do after `rc1`.

Reply:
Created issue: https://github.com/AntelopeIO/leap/issues/783